recipes/boost/all/conanfile.py

```diff
@@ -247,6 +263,28 @@ def config_options(self):
             # Shared builds of numa do not link on Visual Studio due to missing symbols
             self.options.numa = False
 
+        if tools.Version(self.version) >= "1.76.0":
```

Comment:
probably, in future we should add C++ standard requirements to the yml (existing or new), as different libraries tend to change their requirements across boost releases